pom.xml

```diff
@@ -254,7 +254,7 @@
         <dependency>
             <groupId>com.aws.greengrass</groupId>
             <artifactId>nucleus</artifactId>
-            <version>2.1.0-SNAPSHOT</version>
+            <version>2.3.0-SNAPSHOT</version>
```

Comment:
Need logging change in 2.3.0 for tests to pass. Will revert it afterwards

Reply:
No need to revert